hack/fix-EOF.sh

```diff
@@ -30,23 +30,16 @@ FILELIST=($(find . -type f -not \( -path './vendor/*' \
     -o -path './tests/images/*/bin/*' \
     -o -path '*.png' \
     -o -path './tkctl' \
+    -o -path './.idea/*' \
+    -o -path './.DS_Store' \
+    -o -path './*/.DS_Store' \
```

Comment:
Maybe it would be better to get these ignored paths via .gitignore?

Reply:
or we can use `git ls-files` to list files in git repo only